packages/jest-each/package.json

```
@@ -18,6 +18,7 @@
   "license": "MIT",
   "dependencies": {
     "chalk": "^2.0.1",
+    "jest-get-type": "^22.4.3",
```

```suggestion
    "jest-get-type": "^22.1.0",
```

that matches the one in this repo. IDK where 22.4.3 comes from... Missing push after a publish?

---

Not sure about this as it is published on 22.4.3 in npm see: https://www.npmjs.com/package/jest-get-type/v/22.4.3

---

Yeah, I know. But as you can see from the updated lockfile, it's not the one in this repo.

https://github.com/facebook/jest/blob/9cfcd81fd7a3f0b2dc9c004bd53a479035765ae9/packages/jest-get-type/package.json#L4

---

Updated :)